### PR TITLE
0.0.6

### PR DESCRIPTION
**Pull request by mwilko**
Moved MLA data from lifestyle.py. All labels were overlapping and boxes created below the return content syntax. 

Separated  MLA data into respective files, added functions to app.py so the buttons can navigate to and from different screens. MLA seems to update after testing + print statements show the status of the variables.

Edited personal_details.py due to the BMI label updating only after exiting to choice_menu.py then returning. Now the BMI label is updated upon clicking the submit button, which notifies the user the BMI was updated.

### CODE REVIEW

File: src/healthapp/windows/mla_alcohol.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        # main_box.add(noalco_box)
+        # main_box.add(toga.Label(""))
+
+    def alcohol_handler(self, widget):
+        # Update the high blood pressure variable based on the user's selection
+        if widget.text == 'Yes':
+            self.app.user.alcohol = 1
+        elif widget.text == 'No':
+            self.app.user.alcohol = 0
+        print(f"Smoker: {self.app.user.alcohol}")
```
Comment (JaxkDev) [inline src/healthapp/windows/mla_alcohol.py]: typo

File: src/healthapp/windows/mla_diabetes.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      #   main_box.add(nodiabetes_box)
+      #   main_box.add(toga.Label(""))
+
+    def diabetes_handler(self, widget):
+        # Update the high blood pressure variable based on the user's selection
+        if widget.text == 'Yes':
+            self.app.user.stroke = 1
+        elif widget.text == 'No':
+            self.app.user.stroke = 0
+        print(f"Smoker: {self.app.user.diabetes}")
```
Comment (JaxkDev) [inline src/healthapp/windows/mla_diabetes.py]: typo

File: src/healthapp/windows/mla_stroke.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      #   main_box.add(nostroke_box)
+      #   main_box.add(toga.Label(""))
+
+    def stroke_handler(self, widget):
+        # Update the high blood pressure variable based on the user's selection
+        if widget.text == 'Yes':
+            self.app.user.stroke = 1
+        elif widget.text == 'No':
+            self.app.user.stroke = 0
+        print(f"Smoker: {self.app.user.stroke}")
```
Comment (JaxkDev) [inline src/healthapp/windows/mla_stroke.py]: typo

File: src/healthapp/windows/mla_walking.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        # main_box.add(nowalk_box)
+        # main_box.add(toga.Label(""))
+
+    def walking_handler(self, widget):
+        # Update the high blood pressure variable based on the user's selection
+        if widget.text == 'Yes':
+            self.app.user.diffwalking = 1
+        elif widget.text == 'No':
+            self.app.user.diffwalking = 0
+        print(f"Smoker: {self.app.user.diffwalking}")
```
Comment (JaxkDev) [inline src/healthapp/windows/mla_walking.py]: typo

**Review by JaxkDev — APPROVED** [review]
Just typos, happy to merge

File: src/healthapp/windows/mla_alcohol.py (hunk 1 above)
Comment (mwilko) [reply src/healthapp/windows/mla_alcohol.py]: Fixed

File: src/healthapp/windows/mla_diabetes.py (hunk 2 above)
Comment (mwilko) [reply src/healthapp/windows/mla_diabetes.py]: Fixed

File: src/healthapp/windows/mla_stroke.py (hunk 3 above)
Comment (mwilko) [reply src/healthapp/windows/mla_stroke.py]: Fixed

File: src/healthapp/windows/mla_walking.py (hunk 4 above)
Comment (mwilko) [reply src/healthapp/windows/mla_walking.py]: Fixed

**Pull request merged**